### PR TITLE
Additional stream event source mapping properties

### PR DESCRIPTION
**Pull request by jaredstehler**
Adds properties:
* maxBatchingWindowSecs
* bisectBatchOnError
* maxRecordAgeSecs
* maxRetryAttempts
* parallelizationFactor
* tumblingWindowSecs
* destinationConfig { onFailureArn, onSuccessArn }

depends on https://github.com/spinnaker/clouddriver/pull/5319

### CODE REVIEW

File: lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/eventconfig/LambdaUpdateEventConfigurationTask.java [hunk 1]
```diff
@@ -176,7 +176,6 @@ private LambdaUpdateEventConfigurationTaskOutput updateEventConfiguration(Lambda
 
         final List<String> existingEvents = getExistingEvents(lf, targetArn);
         taskInput.getTriggerArns().stream()
-               .filter(curr -> { return !existingEvents.contains(curr); })
```
Comment (jaredstehler) [inline lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/eventconfig/LambdaUpdateEventConfigurationTask.java:179]: this filter blocks updating existing event source mappings; I'm not sure why this was in here?

File: lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/eventconfig/LambdaUpdateEventConfigurationTask.java [hunk 2]
```diff
@@ -192,18 +191,56 @@ private LambdaUpdateEventConfigurationTaskOutput updateEventConfiguration(Lambda
...
+                .maxBatchingWindowSecs(taskInput.getMaxBatchingWindowSecs())
                 .account(taskInput.getAccount()).credentials(taskInput.getCredentials()).appName(taskInput.getAppName())
                 .region(taskInput.getRegion()).functionName(taskInput.getFunctionName()).qualifier(taskInput.getQualifier()).
                         build();
         if (curr.startsWith(DYNAMO_EVENT_PREFIX) || curr.startsWith(KINESIS_EVENT_PREFIX)) {
             if (StringUtils.isNullOrEmpty(taskInput.getStartingPosition())) {
                 taskInput.setStartingPosition(DEFAULT_STARTING_POSITION);
             }
+
+            singleEvent.setBisectBatchOnError(taskInput.getBisectBatchOnError());
```
Comment (shyamsfo) [inline lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/eventconfig/LambdaUpdateEventConfigurationTask.java:203]: Looks good. Did you test these when the testInput does not have these parameters. Does it handle nulls correctly? ARe you sure you dont need the equivalent of StringUtils.isNullOrEmpty around these?

**Review by shyamsfo — COMMENTED** [review]
Looks good. Did you test these when the testInput does not have these parameters. Does it handle nulls correctly? Are you sure you dont need the equivalent of StringUtils.isNullOrEmpty around these?

**Comment by jaredstehler** [comment]
Null values would behave similarly to no input. I think you are right that
we should send null for blank strings, like I had to do for the tumbling
window value. I’ll make that change.

**Comment by jaredstehler** [comment]
updated PR based on comments @shyamsfo

**Review by shyamsfo — APPROVED** [review]

**Pull request merged**